_data/master/navbars/getting-started.yml

```diff
@@ -46,6 +46,11 @@ toc:
   - title: Upgrade
     path: /getting-started/kubernetes/upgrade
 
+- title: OpenShift
+  section:
+    - title: Installation
+      path: /getting-started/openshift/installation
```

Comment:
Sorry, I said I was looking for feedback on general layout, but forgot to mention that Casey and I had already agreed on putting it in it's own orchestrator section for the following reasons:

1. It is different enough from core Kubernetes. For example, it has it's own command line tool - `oc`.
2. This may not be the final home for these docs. Many of these may be moved to openshift documentation upstream.